packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -196,7 +203,7 @@ export function FileUploader({
         ...status,
         uploadTask: uploadTasksTemp?.[index],
         fileState: status.fileState ?? 'loading',
-        percentage: 0,
+        percentage: status.percentage ?? 0,
```

Comment:
This was a bug I found. If you added a file, it uploaded, then added a second file, the first file would be zero'ed out. This saves the percentage